### DISCUSSION TITLE
Practice this week

### DISCUSSION THREAD

**Discussion by coverbeek** [frc5567/2023-Robot#152] [Team Posts]
For practices Monday, Tuesday, and Thursday, we're going to again ask for a reduced crew to allow for the returners (and mentors) to focus in on specific tasks. 

For anyone not invited to the Focused Practice team (you'll get an email from Github if you are invited), please continue to follow the discussions here in your groups and learn and research on your own.

**Comment by coverbeek** [comment]
If you're not sure how to spend your time or what to research, perhaps learning about the motor controllers and related sensors we're using would be a good activity. Courtesy of our friends on team 2767 -- Stryke Force.
https://www.youtube.com/watch?v=VqUPmyUUkEs
https://www.youtube.com/watch?v=cIZORBcM3vg
https://www.youtube.com/watch?v=llPmrsI5zgg
https://www.youtube.com/watch?v=-6Q67sX9rA0
https://www.youtube.com/watch?v=bZLMnFtyiIw
https://www.youtube.com/watch?v=JnOEE89VDo4
https://www.youtube.com/watch?v=F5yFHDHxEVw
https://www.youtube.com/watch?v=Fs46TUeJMyc
https://www.youtube.com/watch?v=D0inPk1Bz7c

**Comment by coverbeek** [comment]
I have reviewed and merged the commits made on Saturday, so master branch is up to date with all commits with Pulls requested. Please be sure to get all of the latest code before making new changes!

**Comment by coverbeek** [comment]
Team, we will stick with focused practice for tomorrow and Thursday and update you on the the plan for Friday, Thursday night. We still have a lot to do and we need to make the best of the time left. Please keep up with the GitHub updates and keep studying on your own time if you’re not asked to be at practice.

**Comment by matthewbogs711** [comment]
Is Saturday another focused practice day, or is everybody needed. I am slightly out of the loop after this week's practices that I wasn't needed at, and I would like to know the plan for Saturday.

**Comment by matthewbogs711** [comment]
I am sending this message again because I haven't gotten a response. What are the plans for practice this weekend? I haven't been needed these last few practices and I am a bit out of the loop because of it. I would highly appreciate it if I could be brought up to speed on this weekend's practices.
Thanks,Matthew Bogoski
    On Tuesday, February 12, 2019, 9:02:02 PM EST, Chris Overbeek <notifications@github.com> wrote:  
 
 
Team, we will stick with focused practice for tomorrow and Thursday and update you on the the plan for Friday, Thursday night. We still have a lot to do and we need to make the best of the time left. Please keep up with the GitHub updates and keep studying on your own time if you’re not asked to be at practice.

**Comment by MikasaSucasa** [comment]
@matthewbogs711 We are still doing focused practice. We do not enjoy limiting the amount of students at practice (we hate it, in fact, since the teaching aspect is the fun part). The issue is that we are currently working on the wiring (yes, we are all still at the school working).  At this point, we may need to further limit participants. There are only so many mentors. Once there is room to breathe and enough free time for the mentors to give attention to all members, we will invite everyone back and go over/ explain every line of code.
